dashboard/src/utils/axiosInstance.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import Cookies from "js-cookie";
+import { showSessionExpired } from "actions/toastActions";
+import store from "store/store";
+
+const { dispatch } = store;
+
+const axiosInstance = axios.create({
+  responseType: "json",
+});
```

Comment:
If you remove the comma from the end of line 9, will Prettier put this all on one line for us?
```
const axiosInstance = axios.create({ responseType: "json" });
```

Reply:
Done!